core/templates/domain/exploration/InteractionObjectFactorySpec.ts

```diff
@@ -656,7 +658,10 @@ describe('Interaction object factory', () => {
     const otherInteraction = iof.createFromBackendDict(otherInteractionDict);
     testInteraction.copy(otherInteraction);
     expect(testInteraction).toEqual(otherInteraction);
-    otherInteraction.customizationArgs.showChoicesInShuffledOrder.value = false;
+    if ('showChoicesInShuffledOrder' in otherInteraction.customizationArgs) {
+      otherInteraction.customizationArgs.showChoicesInShuffledOrder.value =
+        false;
+    }
```

Comment:
Couldn't `!` be used here instead?

Reply:
@vojtechjelinek Can you explain how we could use this?

The error thrown after removing this if condition is:
```
Property 'showChoicesInShuffledOrder' does not exist on type 'InteractionCustomizationArgs'.
  Property 'showChoicesInShuffledOrder' does not exist on type 'AlgebraicExpressionInputCustomizationArgs'.
  ```

Reply:
@mridul-netizen @EeshaArif Cna you help here?

Reply:
@vojtechjelinek Yes this check is required, otherInteraction.customizationArgs is of type `InteractionCustomizationArgs` here which is a union of several other types. It can be the case that property `showChoicesInShuffledOrder` does not exist on these several other types. So we will have to add a check to satisfy type checking.

Reply:
I think a better approach would be to type cast it to the specific type
```suggestion
    const args = (
      <MultipleChoiceInputCustomizationArgs> otherInteraction.customizationArgs
    );
    args.showChoicesInShuffledOrder.value = false;
```

Reply:
done